app/src/main/res/layout/activity_chat.xml

```diff
@@ -5,45 +5,56 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="@color/colorBackground"
-    tools:context="io.left.meshim.activities.ChatActivity"
-    >
+    tools:context="io.left.meshim.activities.ChatActivity" >
 
     <android.support.v7.widget.RecyclerView
         android:id="@+id/reyclerview_message_list"
         android:layout_width="0dp"
-        android:layout_height="0dp"
+        android:layout_height="wrap_content"
         android:layout_marginBottom="8dp"
         android:layout_marginTop="8dp"
         android:scrollbars="vertical"
-        app:layout_constraintBottom_toTopOf="@+id/myMessageEditText"
-        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintBottom_toTopOf="@+id/chat_message_bar"
         app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintTop_toTopOf="parent">
-
-    </android.support.v7.widget.RecyclerView>
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.0"/>
 
-    <EditText
-        android:id="@+id/myMessageEditText"
-        android:layout_width="0dp"
+    <android.support.constraint.ConstraintLayout
```

Comment:
I think this layout is overlapping the recycler view a bit. the time in message is half cut off

Reply:
Fixed. :+1: